### DISCUSSION TITLE
Proper way to change Kibana settings?

### DISCUSSION THREAD

**Discussion by PeterWick** [Security-Onion-Solutions/securityonion#5939] [Unsupported Versions] [answered]
Hello all,
what is the proper way to change Kibana settings? For example I need to change "default route" setting to my edited dashboard, but if I do this in /Kibana/Stack management/Advanced settings after few minutes my changes reverts back. How to make it persistent?
I am using standalone SecurityOnion installation, 2.3.80 version.

Thanks for help.

**Comment by fluffyunicorn24** [comment]
You can try to keep the settings by making the change and export the saved objects with so-kibana-config-export, then place the saved_objects.ndjson in /opt/so/saltstack/local/salt/kibana/files.

Hope this helps!

**Reply by PeterWick** [reply]
Thanks for suggestion, but this doesn't work.

**Comment by notmorpheus** [comment, marked as answer]
Export saved objects from Kibana and place it in `/opt/so/saltstack/local/salt/kibana/files`. You'd also need to convert the ndjson export into jinja since its a salt directory. You can do so by encapsulating the contents of the file with `{% raw -%}` and `{% endraw -%}`. Also make sure to add the `.jinja` extension. The `local` directory actually mirrors the `default` dir and you can see the default `saved_objects` file there in jinja format for reference.

**Reply by PeterWick** [reply]
That works. Thanks for great answer.
It would be interesting to know why changes in Kibana GUI reverts back and we need to do manual works. Do you have an answer?

**Reply by dougburks** [reply]
Kibana configuration is managed with salt:
https://docs.securityonion.net/en/2.3/kibana.html#configuration
